.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.14.0",
-  "packages/api-explorer": "0.9.18",
-  "packages/code-editor": "0.1.9",
-  "packages/extension-api-explorer": "21.12.1",
-  "packages/extension-sdk": "21.12.0",
-  "packages/extension-sdk-react": "21.12.0",
-  "packages/hackathon": "21.12.0",
-  "packages/run-it": "0.9.18",
-  "packages/sdk": "21.12.0",
-  "packages/sdk-codegen": "21.0.24",
-  "packages/sdk-codegen-scripts": "21.0.26",
+  ".": "1.15.0",
+  "packages/api-explorer": "0.9.19",
+  "packages/code-editor": "0.1.10",
+  "packages/extension-api-explorer": "21.13.0",
+  "packages/extension-sdk": "21.12.1",
+  "packages/extension-sdk-react": "21.12.1",
+  "packages/hackathon": "21.13.0",
+  "packages/run-it": "0.9.19",
+  "packages/sdk": "21.12.1",
+  "packages/sdk-codegen": "21.0.25",
+  "packages/sdk-codegen-scripts": "21.0.27",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "21.12.0",
-  "packages/sdk-rtl": "21.0.19",
-  "packages/wholly-sheet": "0.5.15",
+  "packages/sdk-node": "21.12.1",
+  "packages/sdk-rtl": "21.1.0",
```

Comment:
I'd decided to let these `packages/sdk-*` packages start to just semver bump but now the OCD part of my brain can't handle it so I want this to be 21.0.20 - will add the **temporary** release-please-config.json flag